src/gamepad.cpp

```diff
@@ -0,0 +1,374 @@
+#include "gamepad.h"
```

Comment:
I think it would make sense to rename the file (and its corresponding header file) to `sdl_gamepad`.

Reply:
not to me